CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## v1.12.0 (03.03.2021)
```

Comment:
> 03.03.2021

Why did we change the date format from ISO `YYYY-MM-DD` to European `DD.MM.YYYY` for two recent releases?

Reply:
That's a mistake. Thanks for noticing!